### PR TITLE
Add kafka connect

### PR DESCRIPTION
**Pull request by ArtyomMedvedev15**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+services:
+  postgres:
+    image: debezium/postgres:13
+    ports:
+      - 5432:5432
+    healthcheck:
+      test: "pg_isready -U postgres -d mentee"
+      interval: 2s
+      timeout: 20s
+      retries: 10
```
Comment (skalashynski) [inline docker-compose.yml:11]: I do like healthchecks for postgres, can use the same for all services?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    depends_on: [kafka]
+    ports:
+      - 8083:8083
+  schema-registry:
+    image: confluentinc/cp-schema-registry:5.5.3
+    environment:
+     - SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL=zookeeper:2181
+     - SCHEMA_REGISTRY_HOST_NAME=schema-registry
+     - SCHEMA_REGISTRY_LISTENERS=http://schema-registry:8081,http://localhost:8081
+    depends_on: [zookeeper, kafka]
```
Comment (skalashynski) [inline docker-compose.yml]: kafka depends on Zookeper so you can set only `kafka` here.

**Comment by skalashynski** [comment]
add `readme.md` file with description how to run a project

**Review by skalashynski — CHANGES_REQUESTED** [review]

File: initDB.sql [hunk 3]
```diff
@@ -0,0 +1,13 @@
+create schema if not exists department; 
+create table if not exists department.employee
+(
+    id serial primary key,
```
Comment (NevarDima) [inline initDB.sql:4]: I understand that there is no clear technical specification, but think about what constraints you can add to these fields, now you can add a completely empty record to your table

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+services:
+  db:
+    image: 'postgres:13.1-alpine'
+    container_name: db
+    ports:
+      - 5432:5432
+    environment:
+      - POSTGRES_DB=D
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=1234
```
Comment (NevarDima) [inline docker-compose.yml]: How about putting sensitive data (for example a password) into environment variables?

**Pull request merged**